### PR TITLE
Change ui styles

### PR DESCRIPTION
**Pull request by RubashkoM**
- add `progress-spinner`;
- add `card-tertiary`;
- add `btn-tertiary`;
- add `navbar-dark-blue`;
- add `side-navbar`;
- add `vertical-separator`;
- add `icons`;

Сhanged many styles according to the new design and ui-standards.

### CODE REVIEW

File: docs/_includes/ui-standards/colors.html [hunk 1]
```diff
@@ -4,13 +4,15 @@ <h2>Color //</h2>
...
+    <h3 class="mb-4">Primary colors</h3>
+
     <div class="row colors-container">
       <div class="col">
         <div class="color-box">
           <div class="color-square bg-purple"></div>
           <div class="description">
             <small>Purple</small><br>
-            <small>#9012fe</small>
+            <small>#9061af</small>
```
Comment (alex-areshnikov) [inline docs/_includes/ui-standards/colors.html]: This is `$purple` color

File: docs/_includes/ui-standards/colors.html [hunk 2]
```diff
@@ -20,14 +22,13 @@ <h2>Color //</h2>
 
       <div class="col">
         <div class="color-box">
-          <div class="color-square bg-light-blue"></div>
+          <div class="color-square bg-blue"></div>
           <div class="description">
-            <small>Light Blue</small><br>
-            <small>#20d7de</small>
+            <small>Blue</small><br>
+            <small>#477fb2</small>
```
Comment (alex-areshnikov) [inline docs/_includes/ui-standards/colors.html]: This one is `$blue`. Please make sure to define colors in one place and use variables in other places

Comment (RubashkoM) [reply docs/_includes/ui-standards/colors.html]: It's just an example of color hash code. For example, to display `blue` color I use `bg-blue` class, which is includes `background-color: $blue;`.

File: docs/_includes/ui-standards/colors.html (hunk 1 above)
Comment (RubashkoM) [reply docs/_includes/ui-standards/colors.html]: See below.

File: docs/_includes/ui-standards/colors.html (hunk 2 above)
Comment (alex-areshnikov) [reply docs/_includes/ui-standards/colors.html]: Makes sense, sorry about that

**Pull request merged**